securedrop/source_templates/index_locales.html

```diff
@@ -0,0 +1,11 @@
+{% if g.locales|length > 1 %}
```

Comment:
I don't like the idea of the `index_locales`. This should just reuse the other `locales` bit and we can use CSS to make it work in both locations. This feels like unnecessarily duplicating code.